cmake/morpheus_utils/package_config/matx/patches/matx_cccl_interface_fix.patch

```diff
@@ -1,22 +1,18 @@
+From 6ddba385b27f4f7075fcdd5a0916f59944c3ec16 Mon Sep 17 00:00:00 2001
+From: Christopher Harris <xixonia@gmail.com>
+Date: Wed, 7 Aug 2024 21:19:56 +0000
+Subject: [PATCH] fix cccl includes
```

Comment:
Did we ever open an issue with MatX for this?

Reply:
newer versions of matx have fixed this, or at least approach the problem differently due to our speaking up about it.